#!/usr/bin/env python
PACKAGE = "box_dev"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("min_x", int_t, 0, "Minimum X coordinate", 0, 0, 1000)
gen.add("max_x", int_t, 0, "Maximum X coordinate", 100, 0, 5000)
gen.add("min_y", int_t, 0, "Minimum Y coordinate", 0, 0, 1000)
gen.add("max_y", int_t, 0, "Maximum Y coordinate", 100, 0, 5000)

exit(gen.generate(PACKAGE, "elevation_map_adjuster", "Filter"))